src/main/java/seedu/address/model/tut/Tutorial.java

```diff
@@ -207,7 +211,7 @@ public void addTutorialDate(TutDate tutorialDate) {
 
         @Override
         public TutDate getTutorialDate(Date date) {
```

Comment:
Should it return any date here? Or should it return empty date string or placeholder value?